docs/serving/using-knative-services.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "User-created Knative Services"
+weight: 01
+linkTitle: "User-created Knative Services"
+type: "docs"
+---
+
+Knative services are Kubernetes Services that a user creates to deploy an application. Each Knative Service is defined by a Route and a Configuration, which have the same name as the Service, contained in a YAML file.
+
+## Before you begin
+
+To use Knative Services, you will need:
+* A Kubernetes cluster with [Knative installed](https://knative.dev/docs/install/index.html).
+
+## Creating a Knative Service
+To create an application, you need to create a YAML file that defines a Knative Service.
+This YAML file specifies metadata about the application, points to the hosted image of the app for deployment, and allows the deployment to be configured.
```

Comment:
The word "deployment" here might be confusing as it is 
```suggestion
This YAML file specifies metadata about the application, points to the hosted image of the app, and allows the service to be configured.
```

The word deployment here might be confusing as it is overloaded with K8s deployments.